### PR TITLE
[TRIGGER] Wise - Transfer state changed #5456

### PR DESCRIPTION
**Pull request by lcaresia**

### CODE REVIEW

File: components/wise/actions/create-transfer/create-transfer.mjs [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import wise from "../../wise.app.mjs";
+
+export default {
+  name: "Create Transfer",
+  version: "0.0.5",
```
Comment (feyzullah) [inline components/wise/actions/create-transfer/create-transfer.mjs:5]:
```suggestion
  version: "0.0.1",
```

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -8935,7 +8941,7 @@ packages:
     engines: {node: ^8.13.0 || >=10.10.0}
     dependencies:
       '@grpc/proto-loader': 0.6.13
-      '@types/node': 18.0.6
```
Comment (feyzullah) [inline pnpm-lock.yaml:8938]: It seems that `@types/node` is downgraded everywhere it is a dependency. Can you please try to re-run `npx pnpm install -r`?

**Review by feyzullah — COMMENTED** [review]
Hi @lcaresia it seems that this PR and [this one](https://github.com/PipedreamHQ/pipedream/pull/5503) have the same components. Can you please merge these two into a single PR? And I added just two comments. Can you please check?

**Comment by lcaresia** [comment]
@feyzullah I created two PR's because are two task. Just to stay the things more organized.

File: pnpm-lock.yaml (hunk 2 above)
Comment (lcaresia) [reply pnpm-lock.yaml:8938]: I run the command and don't have nothing to commit.

image

**Review by feyzullah — APPROVED** [review]

**Pull request merged**